Promise/Promise+Extras.swift

```diff
@@ -60,7 +76,17 @@ public enum Promises {
             }
         })
     }
-
+    
+    /// Retries a promise for the given amount of times, after the given delay. If the
+    /// given promise does not complete after `count` retries, then the rejection is
+    /// allowed to pass through.
```

Comment:
```suggestion
    /// Retries a promise the given amount of times, after the given delay. If the
    /// given promise does not complete after `count` retries, then the rejection is
    /// passes through.
```

Reply:
CML: `passes` → `passed`

Reply:
Fixed in c9bc94b448931d667b510320c0b65e9096386dd2